akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -43,20 +53,48 @@ sealed trait Multipart {
         .flatMapConcat(ConstantFun.scalaIdentityFunction)
     HttpEntity.Chunked(mediaType withBoundary boundary withCharset charset, chunks)
   }
+
+  /** Java API */
```

Comment:
I mostly meant that we’re not historically used to this scheme: Akka previously only had shared APIs, Streams have completely separate APIs, but HTTP has somewhat related APIs where the inheritance is going in the opposite direction as compared to Actor vs. UntypedActor. Anyway, this is good, we might even patch genjavadoc at some point so that it does not emit the Java API marked methods.